### PR TITLE
PORT-5608 Python Script for Ingesting JFrog Container Images in Port with and keep them updated using Webhook Integration

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description

Added documentation to ingest JFrog artifact, Docker tag and build entities into Port

## Added docs pages

Please also include the path for the added docs

- JFrog (Self-Hosted and Cloud) (`/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+  "relations": {}
+}
+```
+
+</details>
+:::tip Blueprint Properties
+You may modify the properties in your blueprints depending on what you want to track in your JFrog repositories and builds.
+:::
+
+Create the following webhook configuration [using Port UI](../../?operation=ui#configuring-webhook-endpoints)
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md:230]: @lordsarcastic 
1. I'm not sure where this link is supposed to send the reader?
2. From now on please use full path links and not relative ones.

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md [hunk 2]
```diff
@@ -0,0 +1,529 @@
+---
+sidebar_position: 18
+description: Ingest JFrog artifacts, Docker tags and builds into your catalog
+---
+
+# JFrog (Self-Hosted and Cloud)
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md:6]: @lordsarcastic 
The title should be "JFrog", without the (Self-Hosted and Cloud).
If there is a difference between them, please address that in the page where it's relevant

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Comment by hadar-co** [comment]
@lordsarcastic Hi, have you had a chance to look at the comments?

**Comment by lordsarcastic** [comment]
> @lordsarcastic Hi, have you had a chance to look at the comments?

Yes I'll attend to it once I'm done with a pending task

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md (hunk 1 above)
Comment (lordsarcastic) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md:230]: I followed the convention from other documentations as regards creating webhook configurations. Still want me to change it?

Comment (hadar-co) [reply docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md:230]: @lordsarcastic just change the links to be full path and not relative:)

**Comment by hadar-co** [comment]
@lordsarcastic just change the links to be full path and not relative:)

File: docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+  "relations": {}
+}
+```
+
+</details>
+:::tip Blueprint Properties
+You may modify the properties in your blueprints depending on what you want to track in your JFrog repositories and builds.
+:::
+
+Create the following webhook configuration [using Port UI](build-your-software-catalog/sync-data-to-catalog/webhook/?operation=ui#configuring-webhook-endpoints)
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/webhook/examples/jfrog.md:230]:
```suggestion
Create the following webhook configuration [using Port UI](/build-your-software-catalog/sync-data-to-catalog/webhook/?operation=ui#configuring-webhook-endpoints)
```

**Pull request merged**